package.json

```diff
@@ -45,6 +45,8 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.29",
     "@fortawesome/free-solid-svg-icons": "^5.13.1",
     "@fortawesome/react-fontawesome": "^0.1.11",
+    "@openmrs/esm-context": "^3.1.4",
```

Comment:
Context does not exist any more. Also use the 3.1.5 version.

Maybe pull back from master first.

Reply:
Oops, bad rebase. Fixing shortly.